cmd/admin-handlers.go

```diff
@@ -27,6 +28,7 @@ import (
 	"time"
 
 	"github.com/gorilla/mux"
+	"github.com/minio/minio/cmd/logger"
```

Comment:
is there a good reason why we are adding logger as part of `cmd/logger`? we should move this to `pkg/logger/`

Reply:
We have decided to have packages that are useful only to minio to be an internal package

Reply:
If that is the intention then we should move it to cmd/internal/ such that they can never be used externally. But I don't see why it is only internal we have some packages which are directly dependent on server functionality like `pkg/event` which are part of S3 API. Logger doesn't bring such restrictions and i don't see why it should be inside `cmd/` we kept gateway because it is part of `cmd` (as in command) - i don't see why we should have logger as part of `cmd`

Reply:
harsha, AB asked to move minio specific packages to `cmd` directory and have generic packages in `pkg`(as a convention for us) . So we have to move `pkg/event` etc to `cmd/event`

we can have `cmd/internal/PACKAGE` too instead of `cmd/PACKAGE` to be more compatible with Go tooling. We could perhaps do that restructuring in a different commit (i.e moving of gateway code into `internal`, moving of `pkg/event` to `cmd/internal/event`)